drivers/hmis/lib/form_data/default/fragments/current_living_situation.json

```diff
@@ -64,15 +64,15 @@
       ]
     },
     {
-      "type": "STRING",
-      "required": false,
-      "hidden": true,
+      "type": "CHOICE",
+      "required": true,
```

Comment:
Should this field actually be required for CE projects? That was my impression from the ticket https://github.com/open-path/Green-River/issues/6272, but then on the call today we spent some time making sure we enabled the user to null out existing values, so just making sure.

Reply:
Yes looking at the dictionary, it should be required for CE. I still think we should support clearing the value, in case that is desired later on. For example if someone wanted to collect "verified by" in a different project type,  optionally.